django/template/backends/django.py

```diff
@@ -60,7 +61,8 @@ def origin(self):
         return self.template.origin
 
     def render(self, context=None, request=None):
-        context = make_context(context, request)
+        context = make_context(context, request,
```

Comment:
Please keep this on 1 line (we prefer long lines up to 119 characters if it helps readability and avoids non-4 space indent).